lua/methods/coalitions.lua

```diff
@@ -15,5 +15,20 @@ GRPC.methods.getPlayers = function(params)
     result[i] = GRPC.exporters.unit(unit)
   end
   return GRPC.success({units = result})
+end
 
-end
\ No newline at end of file
+GRPC.methods.getGroups = function(params)
+  local result = {}
+  for _, c in pairs(coalition.side) do
+    if params.coalition == nil or params.coalition == c then
```

Comment:
Made `params.coalition` optional

Reply:
It seems wierd that we have a `coalition` service method that does not take a coalition parameter. We are going to break from DCS standards whatever we do here so should we stick with this or maybe move this somewhere else?

Reply:
I agree and updated the implementation in here a5f0eebd40faf87c2885600aa21a7016d963798e